### PR TITLE
Bug 1494374 - make config a bit more generic to support multiple envs

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

**Review by imbstack — APPROVED** [review]

**Comment by djmitche** [comment]
@imbstack I added a commit :)

File: ingress.yaml [hunk 1]
```diff
@@ -2,11 +2,10 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: taskcluster-ingress
-  annotations:
-    kubernetes.io/ingress.class: "nginx"
 spec:
   rules:
-  - http:
+  - host: {$eval: 'root_url[8:]'}
```
Comment (imbstack) [inline ingress.yaml:10]: What's in the first 8 chars?

**Review by imbstack — APPROVED** [review]
This is a really good direction

File: ingress.yaml (hunk 1 above)
Comment (djmitche) [reply ingress.yaml:10]: `h` `t` `t` `p` `s` `:` `/` `/` :)

Comment (djmitche) [reply ingress.yaml:10]: (I added a comment to this effect)

**Pull request merged**